docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -54,6 +54,9 @@ New features
 - Automatically subtract background from runs on loading in EngDiff UI.
 - The most recently created or loaded Calibration is now selected by default in the load path when the interface is opened.
 - The last used RB number is now saved for the next session
+- The generation of the files required for Vanadium normalization is now done on the Focus tab of the user interface. This means the Vanadium data can be updated without
+having to rerun the Ceria calibration. As part of this change the setting "Force Vanadium Recalculation" has been removed and the Vanadium run number input has been
+moved from the Calibration tab to the Focus tab. The Vanadium run number is also no longer written to the prm generated on the Calibration tab
```

Comment:
Should we mention in the release notes that this is a breaking change - i.e. old calibrations cannot be loaded (not a big deal as takes < 1 min to make a new one).

Reply:
I've just tried loading an old .prm file and it doesn't work. Although not because it's trying to load the ceria run number from the file name. The problem occurs in the logic that tries to construct the file names for the calibration .nxs files that are also loaded along with the .prm file.
I think it might be a bit messy to make this backwards compatible because of the fact that the roi can sometimes contain an underscore. So can't really parse from the right hand end of the .prm name to strip off the roi. Instead we need to rely on the left hand section having a fixed number of _'s. So will add a comment to the release note that it's breaking.
While doing this I noticed that the exception handling around the file loading in the calibration model isn't v nice - lots of unhandled exceptions so I'll at least tidy that up